src/main/java/core/basesyntax/service/DataConverterImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    private static final int TYPE_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputReport) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+
+        for (int i = 1; i < inputReport.size(); i++) {

```

Comment:
```suggestion
        for (int i = OFFSET; i < inputReport.size(); i++) {
```
let's make it more visible, + maybe header will take two lines in the future